.github/workflows/ci.yaml

```diff
@@ -1,6 +1,7 @@
 name: CI Build
 
 on:
+  workflow_dispatch:
```

Comment:
Are you sure you want the trailing colon?

Reply:
I tried it and I think you actually do...

Reply:
Yea, I thought it was strange as well but this is how I saw it used in their docs.